docs/run-component-guide.md

```diff
@@ -1,26 +0,0 @@
-## Running the component guide as a standalone app
-
-If you want to work on this gem directly, without having to add it to another
-application, you can run it on your local machine or using [govuk-docker](https://github.com/alphagov/govuk-docker). 
-
-To run it locally you'll need [Ruby](https://www.ruby-lang.org/en/downloads/) and [Node.js](https://nodejs.org/en/) installed to get this project running.
-
-### Running the component guide on your local machine
-```
-cd ~/govuk/govuk_publishing_components
-./startup.sh
-```
-
-The app should start on [http://localhost:3212/component-guide/](http://localhost:3212/component-guide/).
-
-### Running the component guide in [govuk-docker](https://github.com/alphagov/govuk-docker)
-
-```
-cd ~/govuk/govuk-docker
-make govuk_publishing_components
-
-cd ~/govuk/govuk_publishing_components
-govuk-docker-up
-```
-
-Then visit [http://govuk-publishing-components.dev.gov.uk/component-guide/](http://govuk-publishing-components.dev.gov.uk/component-guide/).
```

Comment:
Is this URL listed somewhere else? If not I'm not sure someone new to this would be able to figure it out - or is it listed in the output of the command?

Reply:
Good question, I was being a bit blasé here. This is actually the root URL i.e. going to `govuk-publishing-components.dev.gov.uk` redirects to `/component-guide`. So we don't need to mention it explicitly.